src/zmap.c

```diff
@@ -523,10 +523,14 @@ int main(int argc, char *argv[])
 		zconf.output_filter_str = args.output_filter_arg;
 		log_debug("filter", "will use output filter %s", args.output_filter_arg);
 	}
+	// --ignore-invalid-hosts is deprecated and will be removed in zmap 4.0
+	if (args.ignore_invalid_hosts_given) {
+		log_warn("zmap", "--ignore-invalid-hosts is deprecated and will be removed in zmap 4.0. Use --ignore-blacklist-errors instead to ignore invalid blacklist/whitelist entries.");
+	}
+	zconf.ignore_invalid_hosts = args.ignore_invalid_hosts_given || args.ignore_blacklist_errors_given;
```

Comment:
I considered this, but `ignore_invalid_hosts` is the parameter name used by the blacklist library in all its functions, so this'll be a fairly pervasive change. If that's worth the noise, I'm happy to do it.